apps/btshell/src/cmd.c

```diff
@@ -2712,7 +2712,7 @@ cmd_security_unpair(int argc, char **argv)
 
     rc = parse_dev_addr("peer_", cmd_peer_addr_types, &peer);
     if (rc != 0) {
-        console_printf("invalid 'peer_addr' parameter\n");
+        console_printf("invalid 'peer_addr_type' parameter\n");
```

Comment:
this is not correct. `parse_dev_addr` will parse both `peer_addr` and `peer_addr_type`, depending on values provided